src/functionalTest/java/uk/gov/hmcts/reform/juddata/cameltest/JrdBatchTestValidationTest.java

```diff
@@ -301,5 +304,24 @@ void testChildSkipsOnParentFailure() throws Exception {
         assertEquals(3, auditList.size()); //Personal, Locations, base-locations only
     }
 
+    @Test
+    void testRowIdInExceptionTable() throws Exception {
+
+        uploadBlobs(jrdBlobSupport, archivalFileNames, true, fileWithEmptyPerIdInAuth);
+        uploadBlobs(jrdBlobSupport, archivalFileNames, false, LeafIntegrationTestSupport.file);
+
+        jobLauncherTestUtils.launchJob();
+        validateExceptionDbRecordCount(jdbcTemplate, exceptionQuery, 2, false);
+        List<Map<String, Object>> exceptionList = jdbcTemplate.queryForList(exceptionQuery);
+
+        List<Long> rowId = exceptionList.stream()
+                .map(i -> i.get("row_id"))
+                .map(j -> (Long) j)
+                .collect(Collectors.toList());
+
+        assertTrue(rowId.containsAll(List.of(3L, 5L)));
```

Comment:
I think it is a better practice to use the below for asserting the contents on lists:
```
assertThat(rowId)
            .hasSize(2)
            .hasSameElementsAs(List.of(3L, 5L));
```

Reply:
thanks for the code mate. going forward will use the same for the lists